src/posts/announce-2.3.0.Final.adoc

```diff
@@ -0,0 +1,185 @@
+---
+title: Announcing Thorntail 2.3.0.Final
+publishDate: 2019-1-17
+modifyDate: 2019-1-17
+author: The Thorntail Team
+layout: post.jade
+---
+
+== Thorntail 2.3.0.Final
+
+TODO
+
+=== What's New?
+
+* Upgrade to WildFly 15.0.1
+* New JSONB fraction
+* New EE Security fraction
+* MicroProfile related improvements
```

Comment:
This should state we're now compliant with MP 2.1

Reply:
Done